docs/examples/kubo-as-a-library/go.mod

```diff
@@ -179,6 +179,7 @@ require (
 	github.com/raulk/go-watchdog v1.2.0 // indirect
 	github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572 // indirect
 	github.com/spaolacci/murmur3 v1.1.0 // indirect
+	github.com/stretchr/objx v0.4.0 // indirect
```

Comment:
Needed to avoid a failure in the gotest circleci step that checks whether kubo-as-a-library builds with the current version. Error was 

```
github.com/ipfs/kubo/examples/kubo-as-a-library imports
	github.com/ipfs/kubo/core/coreapi imports
	github.com/ipfs/kubo/tracing imports
	go.opentelemetry.io/otel/exporters/jaeger tested by
	go.opentelemetry.io/otel/exporters/jaeger.test imports
	github.com/stretchr/testify/mock imports
	github.com/stretchr/objx loaded from github.com/stretchr/objx@v0.1.1,
	but go 1.16 would select v0.4.0
```